substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/StackTraceTests.java

```diff
@@ -45,7 +45,7 @@
     public void testGetClassContext() {
         final Subclass sm = new Subclass();
         final Class<?>[] classes = sm.getClassContext();
-        assertSame(StackTraceTests.class, classes[0]);
-        assertTrue(classes.length > 1);
+        assertSame(StackTraceTests.class, classes[1]);
+        assertTrue(classes.length > 2);
```

Comment:
Is this the same as in Java now?

Reply:
Now it's the same. It can be simply tested by starting `db-shootout` benchmark (command is in PR description).

Reply:
We need to run these JUnit tests on the HotSpot VM too as part of the gate runs. 

Also testing more cases would be good.

Reply:
I created a small reproducer for this problem here: https://github.com/jovanstevanovic/DBShootoutException, to confirm the exact same behavior of my changes on both hotspot and ni. 

Agree, we should expand this test.